host_guest/SAMPLing/absolute_submission_example.txt

```diff
@@ -0,0 +1,88 @@
+# SAMPLing challenge results
+#
+# This file will be automatically parsed. It must contain the following five sections: predictions, computational cost,
+# software listing, technical details, and method description. These elements must be provided in the order shown.
+#
+# The data in each prediction line should be a comma-separated list of exactly 201 values, 100 of which are optional.
```

Comment:
The phrasing here "exactly 201 values, 100 of which are optional" is a bit odd -- maybe you mean "exactly 201 fields, 100 of which may or may not be filled" or something similar? Because I don't think an empty field constitutes a "value" so it's unclear how you can have exactly 201 values if some of them might not be there.